tabs/configuration.js

```diff
@@ -96,17 +96,35 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             next_callback();
         }
     }
-    
+
     function load_name() {
-        var next_callback = load_html;
+        var next_callback = load_battery;
         if (CONFIG.flightControllerIdentifier == "BTFL" && semver.gte(CONFIG.flightControllerVersion, "3.0.0")) {
             MSP.send_message(MSPCodes.MSP_NAME, false, false, next_callback);
         } else {
             next_callback();
         }
     }
 
-    
+    function load_battery() {
+        var next_callback = load_current;
+        if (CONFIG.flightControllerIdentifier == "BTFL" && semver.gte(CONFIG.flightControllerVersion, "3.1.0")) {
```

Comment:
The configurator now does a firmware check on connect, and disconnects if non - Betaflight is detected. Hence the `CONFIG.flightControllerIdentifier == "BTFL` check is not needed any more. It just hasn't been removed from all the places.